src/status_im2/navigation/view.cljs

```diff
@@ -69,17 +69,12 @@
         (when js/goog.DEBUG
           [reloader/reload-view])]))))
 
-; Designs require bottom inset to be bigger than safe area, otherwise it is too close to the bottom
-(defn bottom-sheet-insets
-  []
-  (assoc (safe-area/get-insets) :bottom 55))
```

Comment:
wow what a strange code, how it was approved

Reply:
Yeah very strange.. idk